src/main.rs

```diff
@@ -33,12 +33,22 @@ impl Service for HelloWorld {
     type Future = Box<Future<Item=Self::Response, Error=Self::Error>>;
```

Comment:
Instead of hyper::Error use failure::Error. Since it is an executable, it is fine to use an error type which unifies other.